building/tracks/syllabus/README.md

```diff
@@ -0,0 +1,126 @@
+
+## Porting and borrowing
+
+It can be worth looking at how other language tracks have built out their concept exercises.
+That said, if you decide to use other exercises as a starting point for your own, be careful to ensure that the resulting exercise is about the concept as it exists in your language.
+Sometimes concepts differ subtly, sometimes radically.
+Sometimes concepts don't exist at all.
+
+The syllabus, and therefore the concept tree, should represent the concepts that exist in the language.
+
+Don't include concepts just because other tracks do.
+
+In some cases it might be tempting to put a concept in because people often have to work around it by using concepts that do exist.
+Rather than doing this, introduce the concept that the language _does_ use, and consider adding an exercise that explains how to use it in that type of situation.
+
+For example, in Go there are no enums.
+Instead the Go concept tree introduces constants, and teaches how to use constants in the type of situation where you might use enums in other languages.
+
+## Asking for help
+Don't hesitate to ask for help.
+It's better to ask up front or while working on an exercise rather than discussing during code review.
+
+## Getting started
+
+### Developing the first exercise
+
+Rather than trying to map out the entire concept tree up front, just start with the first exercise.
+The goal of the first exercise is to show the student what a small piece of code looks like.
+Most of the code will be provided as a stub.
+The student should only need to make a couple of additions in order to complete the exercise.
+
+Read more about [developing the first exercise](/docs/building/tracks/syllabus/first-exercise.md) here.
+
+### The next exercises
+
+The first exercise should unlock a handful of exercises that introduce fundamental concepts.
+This will be things like primitives or basic types and simple operations on those types.
+
+Read more about [developing the next exercises](/docs/building/tracks/syllabus/next-exercises.md) here.
+
+### And then what?
+
+This is where it often starts getting interesting.
+There is so much you _could_ introduce at this point.
+How do you decide what concepts to tackle next?
+
+It kind of doesn't matter.
+As long as you start somewhere that seems reasonable, it will be fine.
+
+Read more about what we think "reasonable" means in the context of [expanding the concept tree](/docs/building/tracks/syllabus/expanding.md).
+
+## We encourage handwaving
+
+Sometimes you'll feel like there is a deadlock.
+Concept A requires understanding concept B, and B requires understanding A.
+
+In this case simplify.
+Handwave over some complexity in one so that you can get students familiar with the other.
+It's perfectly fine to say that something will be introduced in more depth later, and that for now the student just needs to understand this one bit.
+
+Concepts are understood more deeply in stages and over time.
+
+## Selecting stories
```

Comment:
I think you're right. I'm making a note for now, since I haven't started working on "how to create a concept exercise". This is next on my list, so once I start that I will move some of the things over from this document.